bin/qds.py

```diff
@@ -238,6 +239,7 @@ def _create_cluster_info(arguments, api_version):
                                       aws_availability_zone=arguments.aws_availability_zone,
                                       vpc_id=arguments.vpc_id,
                                       subnet_id=arguments.subnet_id,
+                                      master_elastic_ip=arguments.master_elastic_ip,
```

Comment:
@tanishgupta1 : Is this change required in this PR ? If not, please remove it and raise a separate PR.